Adjust BTP Manager docs to the new documentation requirements (#317)
kyma-project/btp-manager: docs/user/README.md

### SAP BTP Service Operator

SAP BTP Service Operator allows you to connect SAP BTP services to your cluster and then manage them using Kubernetes-native tools.

## Enable BTP Operator module

To enable the BTP Operator module, follow the steps described in the [Install BTP Operator](01-10-installation.md) document.

## How BTP Operator module works

BTP Manager provisions, updates, and deprovisions SAP BTP Service Operator along with its resources, ServiceInstances, and ServiceBindings. SAP BTP Service Operator manages SAP BTP services in your cluster.

Read [BTP Manager operations](../contributor/02-10-operations.md) and the [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator) documentation to learn more.

## Contact and support

For contact and support details, go to {link}.

## Read more

This directory contains the end-user documentation of the BTP Operator module.  

For general information on BTP Manager, see the overarching [documentation](../../docs), and for more details on the BTP Operator module, read the following documents:

> - [Install BTP Operator](01-10-installation.md)

Reviewer: I see this link twice in this document

---

> title: BTP Operator Module

Reviewer 2: ```suggestion
title: BTP Operator module
```
We use [sentence case](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#headings) for headings

---

## Overview

Within the BTP Operator module, [BTP Manager](../../README.md) installs [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator/blob/main/README.md).

## Prerequisites

- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
- Kubernetes cluster (you can use [k3d](https://k3d.io)) 

## Enable BTP Operator module

To enable the BTP Operator module, use the following command:

```
kubectl apply -f https://github.com/kyma-project/btp-manager/releases/latest/download/btp-manager.yaml
```
Use the same command to upgrade the module to the latest version.

For more details on other installation options, read the [Install and uninstall BTP Manager](../contributor/01-01-installation.md) document.

## How BTP Operator module works

> BTP Manager provisions, updates, and deprovisions SAP BTP Service Operator along with its resources, Service Instances, and Service Bindings. SAP BTP Service Operator manages SAP BTP services in your cluster.

Reviewer 2: ```suggestion
BTP Manager provisions, updates, and deprovisions SAP BTP Service Operator along with its resources, ServiceInstances, and ServiceBindings. SAP BTP Service Operator manages SAP BTP services in your cluster.
```

> Read [BTP Manager operations](../contributor/02-01-operations.md) and the [SAP BTP Service Operator](https://github.com/SAP/sap-btp-service-operator) documentation to learn more.

Reviewer 3: I'm wondering if we should link from the `user` folder to the `contributor` folder :thinking:. Theoretically yes, but in practice, wouldn't that confuse the end-user?

Author: I think we can do it as users might want to know more. The link takes them to extra info on the subject. I'll rephrase the sentence to make it clearer.

## Read more

> This directory contains the end-user documentation of the BTP Operator module.  

Reviewer 4: ```suggestion
This directory contains the end user documentation of the BTP Operator module.  
```
